### PR TITLE
CNV-27160_413: Release note deprecated

### PR DESCRIPTION
**Pull request by aspauldi**
Version(s): 4.13

Issue: [CNV-27160](https://issues.redhat.com/browse/CNV-27160)

Link to docs preview: [Deprecated feature in the 4.13 RNs](https://63024--docspreview.netlify.app/openshift-enterprise/latest/virt/virt-4-13-release-notes.html#virt-4-13-deprecated)

QE review:
- [x] QE has approved this change.

### CODE REVIEW

**Comment by aspauldi** [comment]
@dominikholler, Can you please review this update to the Deprecated features for CNV 4.13? Thanks.

File: virt/virt-4-13-release-notes.adoc
Comment (dominikholler) [inline virt/virt-4-13-release-notes.adoc:139]:
```suggestion
* In a future release, support for `virtctl` command line tool installation for {op-system-base-full} 7 and {op-system-base-full} 9 via RPM  will be removed.
```

**Review by dominikholler — CHANGES_REQUESTED** [review]

**Comment by aspauldi** [comment]
@dbasunag, Can you please review this PR? Thanks!

**Review by xenolinux — COMMENTED** [review]
I left a few suggestions but otherwise looks good!

I am new on the peer review squad. Please wait until @abhatt-rh gives my review the go-ahead before implementing any feedback. Thanks!

File: virt/virt-4-13-release-notes.adoc
Comment (abhatt-rh) [inline virt/virt-4-13-release-notes.adoc:136]: Just an alternative suggestion:
```suggestion
Deprecated features are included and supported in the current release. However, they will be removed in a future release and are not recommended for new deployments.
```

File: virt/virt-4-13-release-notes.adoc
Comment (abhatt-rh) [reply virt/virt-4-13-release-notes.adoc:139]: Though it might be clear that the `virtctl` command line tool will be removed in a future release, it is not clear that the `virtctl` command line tool is deprecated in the current release. Consider re-wording for clarity. For more information, see [Deprecated functionality](https://redhat-documentation.github.io/supplementary-style-guide/#_deprecated_functionality) and [Deprecation notice](https://redhat-documentation.github.io/supplementary-style-guide/#_deprecation_notice).

**Review by abhatt-rh — COMMENTED** [review]
Hi @aspauldi and @xenolinux,
I've completed my peer-review follow-up and added a few suggestions for your consideration. Thanks!

**Comment by abhatt-rh** [comment]
Hi @aspauldi  I missed to add that this might need to go through change management since it introduces changes to change to published docs (4.13). You might want to confirm it with your CS/DPM

**Comment by jeana-redhat** [comment]
> Hi @aspauldi I missed to add that this might need to go through change management since it introduces changes to published docs (4.13). You might want to confirm it with your CS/DPM

Seconding this. Can we get confirmation that change management is either done or not required before merging please? 

Otherwise looking fine :slightly_smiling_face: 

/remove-label merge-review-in-progress
/remove-label merge-review-needed

**Comment by jeana-redhat** [comment]
> @jeana-redhat, I've sent a message to the DPM and the CS. I didn't realize that I should wait for their response before asking for a merge. Thanks!

Thanks! Yeah, for off-band support status changes we usually require change management. I don't know if it applies the same to virt, but we need to be sure about that before merging in case it's needed :slightly_smiling_face: Appreciate your patience :bow:

**Comment by aspauldi** [comment]
@jeana-redhat, CNV does not currently have change management in place but might consider it in the future. So, for this PR, change management is n/a. Thanks!

**Comment by apinnick** [comment]
I recommend closing this PR so that we do not need to go through change management. (@aspauldi We usually do have to follow OCP change management.)

**Pull request merged**
